reference/6/Microsoft.PowerShell.Management/Add-Content.md

```diff
@@ -46,66 +46,70 @@ If you need to create files or directories for the following examples, see [New-
 This example appends a value to text files in the current directory but excludes files based on
 their file name.
 
+The **Path** parameter to specifies all `.txt` files in the current directory, but the **Exclude**
```

Comment:
Too many paragraphs. These sentences should be combined into one paragraph.

Also, I would put these two sentences after the code block.